README.md

```diff
@@ -120,6 +120,39 @@ _No codes defined as messages are sent by queue_
 
 **Response Body**
 For debugging purposes, the patched Task is returned.
+  
+#### HTTP GET `/discovery/` (on tapas-auction-house)
+
+**Notes**
+- There is no request body
+- The response's task types are the ones that are sought, i.e. that cannot be executed locally (this way, the crawling auction house should subscribe to the discovered auction house if it can offer that service)
+  
+**Media-Type** `application/auctionhousediscovery+json`
+
+**Response Codes**
+
+- `200` (Accepted) The patch was applied to the Task
+
+**Response Body**
+  
+```json
+  {
```

Comment:
id maybe call it auctionHouseInfo or sth like this, just that the name is a bit easier to interpret

Reply:
Changed the mediatype to `auctionhouseinfo` in 0da0ee6303. Or did you mean the array's name?

Reply:
I think he meant the array name "items". We agreed on AuctionHouseDiscoverInformation for the media type right?

Reply:
yes i meant the array name...thought items is maybe a bit to general? As for the media type we had some more discussions afterwards and agreed on sparing the "information" to keep it a bit shorter. Also JSON is always information...